src/bgw/job.c

```diff
@@ -567,6 +567,148 @@ ts_bgw_job_delete_by_id(int32 job_id)
 	return bgw_job_delete_scan(scankey, job_id);
 }
 
+/* This function only updates the fields modifiable with alter_job. */
+static ScanTupleResult
+bgw_job_tuple_update_by_id(TupleInfo *ti, void *const data)
```

Comment:
These functions make sense to move to the Apache part of the code. Not sure why they were in the TSL part in the first place.

It seems you did not make any changes to the functions when moving them. Is that correct?

Reply:
I did not make any changes, just moved the code around